lib/sinatra/shopify-sinatra-app.rb

```diff
@@ -189,7 +190,9 @@ def self.registered(app)
         provider :shopify,
                  app.settings.api_key,
                  app.settings.shared_secret,
-
+                 #Fixes for new Shopify callback issues
+                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
+                 callback_url: app.settings.hostname + "/auth/shopify/callback",
```

Comment:
Good to know - I more just think that the implementation is kinda sketchy after doing more work with rails generators. It also doesn't really do anything it just copies over and example app and adds a `.env` file. 

By remove I more mean refactor into an example or a cleaner generator